docs/dictionary/operator/wrap.lcdoc

```diff
@@ -45,9 +45,9 @@ iteration ie. 1, 2, 3, 1, 2 . Therefore 5 wrap 3 is 2.
 
 The mathematical formula implemented by the wrap operator is:
 
-    x wraps y        =       ((x-1) mod abs(y)) +1 if (x &gt;= 0)
-
-    =      -((x-1) mod abs(y)) +1 if(x &lt; 0)
+    x wrap y       
+    =       (x-1) mod abs(y) +1     if(x &gt;= 0)
+    =     -((x-1) mod abs(y) +1)    if(x &lt; 0)
 
```

Comment:
According to the engine code:

```
real8 MCU_fwrap(real8 p_x, real8 p_y)
{
	real8 t_y;
	t_y = p_y > 0 ? p_y : -p_y;
	if (p_x >= 0)
		return (fmod(p_x - 1, t_y) + 1);
	else
		return -(fmod(-p_x - 1, t_y) + 1);
		
}
```

So in case of `x<0` this should be:

` =     -((-x-1) mod abs(y) +1)    if(x &lt; 0)`